common-tools/clas-detector/src/main/java/org/jlab/detector/decode/CodaEventDecoder.java

```diff
@@ -963,6 +966,53 @@ public List<DetectorDataDgtz>  getDataEntries_57622(Integer crate, EvioNode node
         return entries;
     }
 
+    /**
+     * Bank TAG=57648 used for DC (Drift Chambers) TDC and ToT values.
+     * @param crate
+     * @param node
+     * @param event
+     * @return
+     */
+    public List<DetectorDataDgtz>  getDataEntries_57648(Integer crate, EvioNode node, EvioDataEvent event){
+        List<DetectorDataDgtz>  entries = new ArrayList<>();
+        if(node.getTag()==57648){
+            try {
+                ByteBuffer     compBuffer = node.getByteData(true);
+                CompositeData  compData = new CompositeData(compBuffer.array(),event.getByteOrder());
+                //List<DataType> cdatatypes = compData.getTypes();
+                List<Object>   cdataitems = compData.getItems();
+
+                int  totalSize = cdataitems.size();
+                int  position  = 0;
+                while( (position + 4) < totalSize){
+                    Byte    slot = (Byte)     cdataitems.get(position);
+                    //Integer trig = (Integer)  cdataitems.get(position+1);
+                    Long    time = (Long)     cdataitems.get(position+2);
+                    Integer nchannels = (Integer) cdataitems.get(position+3);
+                    int counter  = 0;
+                    position = position + 4;
+                    while(counter<nchannels){
+                        Byte   channel = (Byte) cdataitems.get(position);
+                        Short  tdc     = (Short) cdataitems.get(position+1);
+                        Short  tot     = (Short) cdataitems.get(position+2);
+                        position += 3;
+                        counter++;
+                        DetectorDataDgtz   entry = new DetectorDataDgtz(crate,slot,channel);
+                        entry.addTDC(new TDCData(tdc, tot));
+                        entry.setTimeStamp(time);
+                        entries.add(entry);
+                    }
+                }
+            } catch (EvioException ex) {
```

Comment:
Why no exception logging?  Looks like the other bank tags/formats do.

Reply:
added